pelican/plugins/sitemap/test_sitemap.py

```diff
@@ -24,6 +25,7 @@ def tearDown(self):
     def _run_pelican(self, sitemap_format):
         settings = read_settings(
             override={
+                "PATH": BASE_DIR,
```

Comment:
Oh snap, I figured the root cause!

Settings key `CONTENT` isn't used at all I don't think (maybe it was in an old version?), the key is `PATH` here and the default is `"."`.

So if you run the tests from the top-level `sitemap` directory, and you have a `.venv` in the top-level directory (i.e. the case for CI and when I tested), then Pelican recursively reads all the content installed in the virtualenv which includes all the pelican package's test files!

*And* because the `lib64` directory is symlinked to `lib` on Linux, you get two copies of each file which is how this error cropped up when it tried to write the same output file twice

```
❯ find .venv -name first-article.rst
.venv/lib/python3.12/site-packages/pelican/tests/cyclic_intersite_links/first-article.rst

❯ find -L .venv -name first-article.rst
.venv/lib/python3.12/site-packages/pelican/tests/cyclic_intersite_links/first-article.rst
.venv/lib64/python3.12/site-packages/pelican/tests/cyclic_intersite_links/first-article.rst
```

I see the default for `IGNORED_FILES` in Pelican is `.#*` but maybe it should be `.*` to avoid all "hidden" dirs?

Reply:
Regarding the above comment, thanks for your change in https://github.com/getpelican/pelican/pull/3424 that adjusted the `IGNORED_FILES` setting to ignore all hidden files and folders.

Reply:
No worries, glad I was able to help out a little. :grin: